client/my-sites/stats/site.jsx

```diff
@@ -264,6 +267,7 @@ export default connect(
 				siteId
 			),
 			siteId,
+			showChecklistBanner: ! isAtomicSite( state, siteId ) && ! isJetpack,
```

Comment:
`isAtomicSite` is a true subset of `isJetpack`, isn't it? Meaning we could only use `isJetpack` as the criterion?

Reply:
For now, that's true and this is redundant. However, the cost is _extremely low_, and I like forcing future devs to consider Atomic.